docker/libs/stats/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM opendatacube/geobase:wheels as env_builder
+
+ARG py_env_path=/env
+
+ENV LC_ALL=C.UTF-8
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Do the apt install process, including more recent Postgres/PostGIS
+RUN apt-get update && apt-get install -y wget gnupg \
+    && rm -rf /var/lib/apt/lists/*
+RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | \
+    apt-key add - \
+    && echo "deb http://apt.postgresql.org/pub/repos/apt/ bionic-pgdg main" \
+    >> /etc/apt/sources.list.d/postgresql.list
```

Comment:
Can we please test without this step as I don't think it's needed.

Reply:
It worked without these.  I've deleted these.